pkg/Makefile

```diff
@@ -35,7 +58,12 @@ deb: prep
 		--url $(URL) \
 		--vendor $(VENDOR) \
 		--architecture $(DEB_PKG_ARCH) \
-		--package $(TARGET_DIR)/aerospike-prometheus-exporter-$(VERSION)-$(DEB_PKG_ARCH).deb
+		--package $(TARGET_DIR)/$(PKG_FILENAME)_$(VERSION)_$(DEB_PKG_ARCH).deb
```

Comment:
should use hyphens (-) instead of underscores (_). 
naming should be backward compatible and consistent with the rpm style too.